src/utils/_Popper.js

```diff
@@ -88,6 +88,7 @@ class Popper extends React.Component {
 
         let popper = (
             <ReactPopper
+                {...popperProps}
```

Comment:
`innerRef` could not be set without the props going to this level because it only exists in `react-popper`.

Reply:
We might need an additional prop then to spread to the inner div

Reply:
I decided to just separate the ref out from the `popperProps`. Don't think there would be a need to have two sets of props to spread.